submissions/nadsatt/friends-app/modules/page-link-list.js

```diff
@@ -0,0 +1,151 @@
+import { PageLink, CornerPageLink } from './page-link-items.js';
+
+export class PageLinkList {
+    constructor(userService, userCardList){
+        this.userService = userService;
+        this.userCardList = userCardList;
+        this.fragment = new DocumentFragment();
+
+        this.usersPerPage = 10;
+        this.maxPageLinksNumber = 5;
+
+        this.defineCornerPageLinks();
+        this.defineElement();
+        this.insertElement();
+        this.defineElementClickHandler();
+    }
+
+    defineCornerPageLinks(){
+        this.leftPageLinks = ['First', 'Prev'].map(value => new CornerPageLink(value).element);
+        this.rightPageLinks = ['Next', 'Last'].map(value => new CornerPageLink(value).element);
+    }
+
+    defineElement(){
+        this.element = document.createElement('ul');
+        this.element.classList.add('page-link-list');
+    }
+
+    insertElement(){
+        document.querySelector('.page-link-list-wrapper').append(this.element);
+    }
+
+    defineElementClickHandler(){
+        this.element.onclick = ({target, target:{textContent: value}}) => {
+            if(this.checkIfMiddlePageLinkClicked(target)){
+                let currentPageNumber = +value;
+                this.performPagination(currentPageNumber);
+            }
+            else if(this.checkIfCornerPageLinkClicked(target)){
+                let currentPageNumber = value === 'First' ? 1 :
```

Comment:
1. It's a `const`
2. Nested ternary go brrrrr :)

Can you replace this? As an example, you can use map collection. I can show you how if you want.